test/functional/notes_controller_test.rb

```diff
@@ -235,45 +234,6 @@ def setup
 
   #end
 
-  test "edit note" do
-    UserSession.create(rusers(:bob))
-    title = "My second post about balloon mapping"
-
-    post :create, 
-         title: title, 
-         body: "This is a fascinating post about a balloon mapping event.", 
-         tags: "balloon-mapping,event"
-         #, main_image: "/images/testimage.jpg"
-
-    assert_redirected_to "/notes/"+rusers(:bob).username+"/"+Time.now.strftime("%m-%d-%Y")+"/"+title.parameterize
-
-    node = DrupalNode.where(title: title).first
-
-    # add a tag, and change the title and body
-    newtitle = title + " which I amended"
-
-    post :update, 
-         id: node.id, 
-         title: newtitle, 
-         body: "This is a fascinating post about a balloon mapping event. <span id='teststring'>added content</span>", 
-         tags: "balloon-mapping,event,meetup"
-
-    assert_redirected_to "/notes/" + rusers(:bob).username + "/" + Time.now.strftime("%m-%d-%Y") + "/" + title.parameterize + "?_=" + Time.now.to_i.to_s
-
-    # approve the first-timer's note:
-    node.publish
-
-    get :show, 
-        author: rusers(:bob).username, 
-        date: Time.now.strftime("%m-%d-%Y"), 
-        id: title.parameterize
-
-    assert_equal flash[:notice], "Edits saved."
-    assert_select "h1", newtitle
-    # assert_select "span#teststring", "added content" # this test does not work!! very frustrating. 
-    # assert_select ".label", "meetup" # test for tag addition too, later
-  end
-
```

Comment:
Moved this part to integration tests since this test involves multiple requests. Functional tests are best for handling single requests since the request variable doesn't update with  new requests. If we wan't to test more than one request in a test we should use integration test.